PS-9017: Check OpenSSL FIPS mode status during server startup (8.0.35) (#5175)
percona/percona-server: share/messages_to_error_log.txt

ER_LOG_NAME_NOT_MATCHING_SEC_LOG_PATH
  eng "The --secure-file-path is configured, %s must be set accordingly."

ER_SSL_FIPS_MODE_ENABLED

>   eng "We have detected a FIPS-approved version of OpenSSL library in your system. It has the default crypto provider set to use FIPS mode. Consequently Percona Server for MySQL will run in FIPS mode as well."

Reviewer: ```suggestion
  eng "A FIPS-approved version of OpenSSL cryptographic library has been detected in the operating system. It has a properly configured FIPS module available for loading. Consequently Percona Server for MySQL will run in FIPS mode as well."
```

Author: Done

>   eng "A FIPS-approved version of OpenSSL cryptographic library has been detected in the operating system. It has a properly configured FIPS module available for loading. Consequently Percona Server for MySQL will run in FIPS mode as well."

Reviewer: ```suggestion
  eng "A FIPS-approved version of the OpenSSL cryptographic library has been detected in the operating system with a properly configured FIPS module available for loading. Percona Server for MySQL will load this module and run in FIPS mode."
```
Approved by @patrickbirch

Author: Updated
